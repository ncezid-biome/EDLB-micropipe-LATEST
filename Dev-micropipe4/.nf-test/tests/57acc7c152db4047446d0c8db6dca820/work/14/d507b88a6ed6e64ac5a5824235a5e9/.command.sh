#!/bin/bash -ue
set +eu
medaka_consensus -i filtered.fastq.gz -d flye_racon_4.fasta -o $PWD -t 8 -m r941_min_sup_g507
rm consensus_probs.hdf calls_to_draft.bam calls_to_draft.bam.bai
cp .command.log medaka.log
medaka --version > medaka_version.txt
