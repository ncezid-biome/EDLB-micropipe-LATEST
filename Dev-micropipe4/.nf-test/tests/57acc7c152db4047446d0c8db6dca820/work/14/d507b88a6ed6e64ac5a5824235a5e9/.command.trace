nextflow.trace/v2
realtime=26000
%cpu=1538
rchar=281943127
wchar=24362949
syscr=48705
syscw=2302
read_bytes=241592320
write_bytes=19365888
%mem=3
vmem=5895668
rss=449820
peak_vmem=5895672
peak_rss=449820
vol_ctxt=507
inv_ctxt=79
