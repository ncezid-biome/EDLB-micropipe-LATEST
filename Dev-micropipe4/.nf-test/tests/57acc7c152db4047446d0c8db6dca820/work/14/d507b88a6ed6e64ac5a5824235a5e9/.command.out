Checking program versions
This is medaka 1.6.0
Program    Version    Required   Pass     
bcftools   1.15       1.11       True     
bgzip      1.15       1.11       True     
minimap2   2.24       2.11       True     
samtools   1.15       1.11       True     
tabix      1.15       1.11       True     
Warning: Output /scicomp/home-pure/suj7/Dev-micropipe4/.nf-test/tests/57acc7c152db4047446d0c8db6dca820/work/14/d507b88a6ed6e64ac5a5824235a5e9 already exists, may use old results.
Aligning basecalls to draft
Running medaka consensus
Polished assembly written to /scicomp/home-pure/suj7/Dev-micropipe4/.nf-test/tests/57acc7c152db4047446d0c8db6dca820/work/14/d507b88a6ed6e64ac5a5824235a5e9/consensus.fasta, have a nice day.
